recipe-server/normandy/settings.py

```diff
@@ -389,6 +398,11 @@ class Production(Base):
     DEFAULT_FILE_STORAGE = values.Value('normandy.base.storage.S3Boto3PermissiveStorage')
     AWS_S3_FILE_OVERWRITE = False
 
+    # Custom CORS settings that overrides the CORS class's configuration.
+    # In production we harden it down a bit extra.
+    CORS_ORIGIN_ALLOW_ALL = values.BooleanValue(False)
+    CORS_ORIGIN_WHITELIST = values.ListValue([])
```

Comment:
We should include this value in `Base` (or rather in `Cors`), so that other environments can override it with envvars too. For example, I can't set `DJANGO_CORS_ORIGIN_WHITELIST` in local development, because the `Development` class doesn't read the envvar.